drift/inventory_service_interface.py

```diff
@@ -1,9 +1,11 @@
 import json
 import os
 import requests
+from flask import current_app
```

Comment:
Is there a way to pass around the logger without requiring 'flask` ? This had a nice encapsulation before being loosely coupled.

Reply:
ya, I could send it into `fetch_systems`. The other option is to not log what the result is and assume we can just look at the logs on the other service and correlate with timestamps.